SamLau95/nbinteract
# Implement bqplot rendering

Resolves #5.

This PR enables bqplot widgets to render in HTML pages generated by nbinteract.

![screenshot 2017-12-20 23 43 51](https://user-images.githubusercontent.com/2468904/34245649-b517dba8-e5df-11e7-91c4-4148bfe44e53.png)

**How to test:**

1. Run `make serve -j2`
2. Open http://localhost:8080/bqplot_ols_demo.html
3. Verify that interactive plots show up
